presto-parquet/src/main/java/com/facebook/presto/parquet/reader/PrimitiveColumnReader.java

```diff
@@ -272,11 +274,13 @@ private ValuesReader readPageV1(DataPageV1 page)
         definitionReader = new LevelValuesReader(dlReader);
         try {
             byte[] bytes = page.getSlice().getBytes();
-            rlReader.initFromPage(page.getValueCount(), bytes, 0);
-            int offset = rlReader.getNextOffset();
-            dlReader.initFromPage(page.getValueCount(), bytes, offset);
-            offset = dlReader.getNextOffset();
-            return initDataReader(page.getValueEncoding(), bytes, offset, page.getValueCount());
+            final ByteBufferInputStream bufferInputStream = ByteBufferInputStream.wrap(ByteBuffer.wrap(bytes));
```

Comment:
remove final modifiers.

Reply:
Also I think we should rename the `rl` and `dl` to their full form as it's not easy to read for people who are not familiar with Parquet.

Reply:
Both done.